### PR TITLE
Trackers: seoul.co.kr

### PR DESCRIPTION
**Pull request by ghost**
### URL: `https://www.seoul.co.kr/news/newsView.php?id=20201217005002`
Screenshot:
Screenshot

### Network Logs
network1
network2
network3
network4

### Code Preview
```
https://www.seoul.co.kr/weblog/proc/weblog_seoul.js
https://www.seoul.co.kr/weblog/proc/weblog_seoul_google.js
```
debugger1
debugger2

### CODE REVIEW

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -22,6 +22,8 @@
 ||nd.demdex.net/event?d_mid=$redirect=nooptext,important,domain=9now.com.au
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 !
+||seoul.co.kr/weblog/proc/weblog_*.php
+||seoul.co.kr/weblog/proc/weblog_seoul*.js
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:26]: why not `||seoul.co.kr/weblog/proc/weblog_`?

Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: Okay, b4cd5138ef600e1e9a31388f6e296c538671e410

Comment (Yuki2718) [reply SpywareFilter/sections/specific.txt]: Or even `||seoul.co.kr/weblog/`? I see this is in EasyPrivacy. Can this cause trouble?

Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: 4d1124c0ff60dc7ad3cc205110053a78539323d4

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**